src/cloudFlowUtils/computeWiredGraph.ts

```diff
@@ -12,26 +12,27 @@ import { RawSubnetData, Subnet, SubnetData } from "./models"
 export function computeWiredGraph(data: RawSubnetData): SubnetData {
   const { vertices, edges: links } = data
 
-  const subnets = [...Object.values(vertices)].flatMap(v => [
-    {
-      id: `SOURCE_${v.id}`,
-      account: `SOURCE_${v.account}`,
-      region: `SOURCE_${v.region}`,
-      vpc: `SOURCE_${v.vpc}`,
-      az: `SOURCE_${v.az}`,
-      subnet: `SOURCE_${v.subnet}`,
-      isTarget: false,
-    },
-    {
-      id: `TARGET_${v.id}`,
-      account: `TARGET_${v.account}`,
-      region: `TARGET_${v.region}`,
-      vpc: `TARGET_${v.vpc}`,
-      az: `TARGET_${v.az}`,
-      subnet: `TARGET_${v.subnet}`,
-      isTarget: true,
-    },
-  ])
+  const subnets = [...Object.values(vertices)].flatMap(v => {
+    let prefix = "SOURCE_"
+    if (v.isTarget) {
+      prefix = "TARGET_"
```

Comment:
Can you explain this part a bit more? It seems to me that the raw data now contains the `isTarget` property... but does that mean the backend will send the payload containing both? Or does it mean we will add the `isTarget` property after receiving from the backend?

From the backend's perspective, I believe they would just send a single representation, and we only need to split `source`/`target` for presentational purposes (otherwise the graph would be cyclical/not work for UI representation properly)